book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+end
```

Comment:
- [ ] Please kindly ensure the `many-to-many` (also `has-many-through`) relationship between `Person` and `Book` is created using the intermediate class `Rental` by creating an `add_rental` method.
- [ ] Also, please ensure the `rental` has a getter and setter.
Hint:
```ruby
def add_rental(person, date)
    Rental.new(date, self, person)
  end
```

Reply:
Thank you